### PR TITLE
Feature/finish callback

### PR DESCRIPTION
**Pull request by maxkomarychev**
Add a callback prop that is called when animation is finished.

partially relevant to https://github.com/airbnb/lottie-react-native/issues/3

### CODE REVIEW

**Comment by josephphayden** [comment]
What's the status of this? Very keen for this feature :)

File: lib/android/src/main/java/com/airbnb/android/react/lottie/ReactLottieAnimationView.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import android.animation.Animator;
+import android.content.Context;
+
+import com.airbnb.lottie.LottieAnimationView;
+import com.facebook.react.bridge.Arguments;
+import com.facebook.react.bridge.ReactContext;
+import com.facebook.react.bridge.WritableMap;
+import com.facebook.react.uimanager.events.RCTEventEmitter;
+
+public class ReactLottieAnimationView extends LottieAnimationView implements Animator.AnimatorListener {
```
Comment (varunrau) [inline lib/android/src/main/java/com/airbnb/android/react/lottie/ReactLottieAnimationView.java]: why is this class necessary? why can't `LottieAnimationView` implement `Animator.AnimatorListener`

**Comment by luisfuertes** [comment]
lottie-react-native already has this feature?

**Comment by gpeal** [comment]
This will need to be augmented in a few ways to handle pause, cancel, repeat, and other events as well as the merge conflicts since this. Since this is now pretty old, I'm going to close this particular PR but either you can put up a new one or I'll work on this soon.

**Comment by maxkomarychev** [comment]
I ended up using simple wrapper with `Animated`-driven animations. Proposed solution is no longer relevant and in fact looks too complicated.

**Pull request closed without merging**